fuse/index.d.ts

```diff
@@ -3,22 +3,20 @@
 // Definitions by: Greg Smith <https://github.com/smrq/>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-declare class Fuse {
-	constructor(list: any[], options?: fuse.IFuseOptions);
-	search(pattern: string): any[];
+declare class Fuse<T> {
+	constructor(list: T[], options?: Fuse.Options<T>);
+	search(pattern: string): T[];
 }
 
-declare namespace fuse {
+declare namespace Fuse {
 	interface IFuseOptions {
 		id?: string;
 		caseSensitive?: boolean;
 		include?: string[];
 		shouldSort?: boolean;
-		searchFn?: any;
```

Comment:
@andy-ms , this was a mistake from my side, fixed now..
and here is the reference for change -> [link](https://github.com/krisk/Fuse#options)